static/js/people.js

```diff
@@ -69,7 +69,7 @@ exports.get_by_email = function (email) {
 
 exports.get_realm_count = function () {
     // This returns the number of active people in our realm.  It should
```

Comment:
If it also includes bots, then saying that it returns the number of active people (only) is incorrect, right?